Demo/Common/IntegrationExamples/DropIn.swift

```diff
@@ -0,0 +1,121 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import AdyenActions
+import AdyenCard
+import AdyenComponents
+import AdyenDropIn
+import AdyenNetworking
+import AdyenSession
+import UIKit
+
+internal final class DropIn: DropInExampleProtocol {
+
+    // MARK: - Properties
+
+    internal var dropInComponent: PresentableComponent?
+    internal weak var presenter: PresenterExampleProtocol?
+    internal var paymentMethods: PaymentMethods?
```

Comment:
We don't really need this - `session?.paymentMethods` could be used instead

Reply:
Yes, your are right. I have removed this and using ```session?.paymentMethods``` instead. https://github.com/Adyen/adyen-ios/pull/1184